protocol/src/main.rs

```diff
@@ -0,0 +1,234 @@
+// Copyright 2023 Oxide Computer Company
+
+use std::net::SocketAddr;
+use std::time::Duration;
+use std::time::Instant;
+
+use crucible_common::Block;
+use crucible_protocol::BlockContext;
+use crucible_protocol::CrucibleDecoder;
+use crucible_protocol::CrucibleEncoder;
+use crucible_protocol::EncryptionContext;
+use crucible_protocol::JobId;
+use crucible_protocol::Message;
+
+use anyhow::Result;
+use clap::Parser;
+use futures::SinkExt;
+use futures::StreamExt;
+use tokio::net::TcpListener;
+use tokio::net::TcpSocket;
+use tokio_util::codec::FramedRead;
+use tokio_util::codec::FramedWrite;
+use uuid::Uuid;
+
+#[derive(Debug, Parser)]
+#[clap(about = "Protocol serialization and deserialization speed test")]
+enum Args {
+    Dynamometer {
+        #[clap(short, long, default_value_t = 512)]
```

Comment:
This ones missing the doc comment.

Reply:
The doc comment should also specify the unit (it looks like bytes)